lib/sensors/src/MoistureSensor.cpp

```diff
@@ -15,6 +15,11 @@ constexpr int Sensor_Status_Reset      = 0x7F;
 
 MoistureSensor::MoistureSensor(PinName sda, PinName scl) : i2c_(sda, scl) {}
 
+/*
+** NOTE FROM YEHIA, I AM UNSURE FOR THE TIMING FUNCTIONS IN THIS CODE PREVIOUS CODE DIDNT HAVE ANY
+UNITS FOR THE SLEEP FUNCTIONS AND SO I GUESSED EVERYTHING IN MS**
+*/
+
```

Comment:
look at @FK3wong PR for sensor rearchitecture. just use his changes for this file and remove comments abt uncertainity.

Reply:
done